.github/workflows/test-coding-standards.yml

```diff
@@ -58,4 +58,4 @@ jobs:
         run: composer update --ansi --no-interaction
 
       - name: Run lint
-        run: composer cs
+        run: PHP_CS_FIXER_IGNORE_ENV=1 composer cs
```

Comment:
```suggestion
        run: composer cs
        env:
            PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.php-version == "8.4" }}
```

Reply:
Sorry, the syntax was not correct. It should be single quotes, not double

Reply:
Rebased again